builder/azure/arm/template_factory.go

```diff
@@ -8,6 +8,8 @@ import (
 
 	"github.com/mitchellh/packer/builder/azure/common/constants"
 	"github.com/mitchellh/packer/builder/azure/common/template"
+
+	_ "github.com/approvals/go-approval-tests"
```

Comment:
hmm, I suppose we could manually add it to vendor.json (or do nothing as it's in there already) and take out this line.

I don't see any govendor utilities for pulling in tests